prog/dftb+/lib_dftb/dftbplusu.F90

```diff
@@ -16,15 +16,80 @@ module dftbp_dftbplusu
   use dftbp_fileid
   use dftbp_commontypes
   use dftbp_spin
+  use dftbp_sorting, only : heap_sort
   implicit none
   private
 
-  public :: getDftbUShift, AppendBlock_reduce, Block_expand
-  public :: E_DFTBU, DFTBplsU_getOrbitalEquiv, DFTBU_blockIndx
-  public :: plusUFunctionals
+  public :: TDftbU_inp, TDftbU, TDftbU_init, plusUFunctionals, AppendBlock_reduce
 
-  
-  !> Contains functional characteristics
+  !> Input for DFTB+U calculation
+  type TDftbU_inp
```

Comment:
I'd favour `TDftbInp` or `TDftbInput`

Reply:
How about TDftb**U**Inp ?

Reply:
Yes, of course, it was just a typo. I meant `TDftbUInp` or `TDftbUInput'.